README.md

```diff
@@ -9,22 +9,56 @@
 [![build](https://img.shields.io/github/workflow/status/exadel-inc/esl/validate/main?style=for-the-badge)](https://github.com/exadel-inc/esl/actions/workflows/validate.yml)
 [![License](https://img.shields.io/badge/license-MIT-green.svg?style=for-the-badge)](./README.md)
 
-Exadel Smart Library (**ESL**) is a web components based library that gives you a set of **lightweight** and **flexible** custom elements to easily create basic UX modules and make your websites super fast.
+Exadel Smart Library (**ESL**) is an open-source **lightweight** and **flexible** UI component library based on web components.
+Build UX modules with custom components provided by our library and make sites work faster. 
+ESL web UI components library is compatible with any framework which makes ESL ideal for projects regardless of 
+their tech stack.
 
 <p align="center">
-<a href="https://esl-ui.com"><img src="./docs/images/btn.png" alt="Project Site / Live Examples" width="350" height="65" title="Project Site / Live Examples"/></a>
+<a href="https://esl-ui.com"><img src="./docs/images/welcome-btn.png" alt="Visit our UI component library website with examples" width="409" height="75" title="Click to visit our UI component library website with examples"/></a>
 </p>
 
-<p align="center" >★ <b>Try out our open source library and, if you like it, please support us with a star on GitHub</b> ★</p>
+<p align="center" >★ <b>Check out our UI component library and support it with a star</b> ★</p>
+
+- [Overview](#overview)
+- [Library Structure](#library-structure)
+- [Installation Guide](#installation-guide)
+- [Roadmap](#roadmap)
+- [Development Information for Contributors](#development-information-for-contributors)
+
+## Overview
+
+Exadel Smart Library, or ESL for short, is a free web UI components library that reduces routine tasks and lets 
+you efficiently create websites with multiple web components at hand. 
+While looking for the best UI component library, keep in mind that it should be compatible with other libraries 
+and frameworks. And ESL is!
```

Comment:
```suggestion
and frameworks. And ESL does!
```

Reply:
It looks like the original one is more proper here.